README.rst

```diff
@@ -24,14 +24,64 @@ What is Barril?
 
 Python package to manage units for physical quantities.
 
+Quick example:
 
-* Free software: MIT license
+.. code-block:: python
+
+    from barril.units import Scalar
+
+    s1 = Scalar(10, 'm')
+    s2 = Scalar(500, 'cm')
+    assert s1 + s2 == Scalar(15, 'm')
 
 
 Features
 --------
 
-* TODO
+* Pre-defined unit database containing several physical quantities for the Oil & Gas industry.
```

Comment:
I think we should target scientific community as a hole. Having "Oil & Gas industry" may give the wrong impression